Core/Systems/Affixes/ItemTypes/ModifyHitAffixes.cs

```diff
@@ -0,0 +1,60 @@
+﻿using PathOfTerraria.Content.Buffs;
+using Terraria.ID;
+
+namespace PathOfTerraria.Core.Systems.Affixes.ItemTypes;
+
+internal class PiercingItemAffix : ItemAffix
+{
+	public override void ApplyAffix(EntityModifier modifier, PoTItem gear)
+	{
+		modifier.ArmorPenetration.Base += Value;
+	}
+}
+
+internal class AddedKnockbackItemAffix : ItemAffix
+{
+	public override void ApplyAffix(EntityModifier modifier, PoTItem gear)
+	{
+		modifier.Knockback += Value;
```

Comment:
This should also have /100, yeah?

Reply:
We could probably change the class to divide the input value when you use + operator by 100 automatically, might consider later?